Sources/Paywalls/Components/Common/PaywallComponentPropertyTypes.swift

```diff
@@ -55,6 +55,13 @@ public extension PaywallComponent {
 
     }
 
+    enum Shape: Codable, Sendable, Hashable, Equatable {
+
+        case rectangle
+        case pill
```

Comment:
I see that `Shape` and `CornerRadiuses` are used together, but as separate things. I feel like corner radiuses are a property of the shape. For instance, a `pill` is a `rectangle` with 50% rounded corners.  

Maybe we can give `rectangle` a `CornerRadiuses` property, and remove `pill`? Or maybe we don't need `Shape` at all, if all shapes are variations in corner radiuses.

Reply:
Or the new `RadiusInfo` from https://github.com/RevenueCat/purchases-ios/pull/4328 😅